installer/x86_64/install.sh

```diff
@@ -65,9 +65,9 @@ fi
 echo "onie_platform: $onie_platform"
 
 # default console settings
-CONSOLE_PORT=0x3f8
-CONSOLE_DEV=0
-CONSOLE_SPEED=9600
+CONSOLE_PORT=0x2f8
+CONSOLE_DEV=1
+CONSOLE_SPEED=115200
```

Comment:
please do not modify this file.

Reply:
OK, but we found the issue that we can't access SONiC console after we install it with this old config